reference/progress.v1.yaml

```diff
@@ -312,316 +162,18 @@ paths:
       description: Get simple progress statements for a specific user
 components:
   schemas:
-    Statement:
-      title: XAPI Statement
+    SimpleProgress:
+      title: SimpleProgress
       type: object
       x-examples:
         example-1:
           id: string
-          actor:
-            $ref: '#/components/schemas/Agent'
-          verb: {}
-          object:
-            $ref: '#/components/schemas/Activity'
-          result:
-            score:
-              scaled: 0
-              raw: 0
-              min: 0
-              max: 0
-            success: true
-            completion: true
-            response: string
-            duration: string
-            extensions: {}
-          context: {}
+          activityId: string
```

Comment:
Should we use AssignableUnitId here, to make clear that this is the ID of the AU?